src/test/java/hudson/plugins/logparser/LogParserTestCase.java

```diff
@@ -84,7 +85,7 @@ protected WorkflowJob setupWorkflowProject(boolean broken) throws IOException {
             // until the block is done, they are in the numbered log filed for each workflow step though
             + "node { \n"
             + "  writeFile file: 'temp.rules', text: \"\"\"error /ERROR/\nwarn /WARN/\ninfo /INFO/\n\"\"\"\n"
-            + "  step([$class: 'LogParserPublisher'" + (broken ? "" : ", failBuildOnError: true, unstableOnWarning: true, showGraphs: true, useProjectRule: true, projectRulePath: 'temp.rules'") + "])\n"
+            + "  step([$class: 'LogParserPublisher'" + (broken ? "" : ", failBuildOnError: true, unstableOnWarning: true, showGraphs: true, projectRulePath: 'temp.rules'") + "])\n"
```

Comment:
This is to simplify the notation as `useProjectRule` is well defined by using one of the parameters `projectRulePath`, `parsingRulesPath`.